src/NumericalAlgorithms/Spectral/SwshTransform.hpp

```diff
@@ -221,11 +221,12 @@ class TransformJob {
 
 namespace detail {
 // helper function for libsharp transforms which adds to a vector of pointers
-// the set of pointers associated with angular views of the provided collocation
-// point vector. This function will perform a conjugate on the collocations if
-// `positive_spin` is false, which will need to be reversed by the end of the
-// transform routine to restore the original values by the end of the transform
-// procedure.
+// `collocation_data` the set of pointers associated with angular views of the
+// provided collocation data `vector`. The views are stored in
+// `collocation_views`. This function will perform a conjugate on the
```

Comment:
"if `positive_spin` is false, this function will conjugate the views, and therefore potentially conjugate (depending on `Representation`) the input data `vectors`. This behavior is chosen to avoid frequent copies of the input data `vector` for optimization. The conjugation must be undone after the call to libsharp transforms using `FUNCTION NAME` in order to return the input data to its original state."
and add `FUNCTION NAME`